src/chapters/saml-auth.xml

```diff
@@ -0,0 +1,183 @@
+                            information the SAML extension needs in order to
+                            know how to authenticate with the IdP. This URI can
+                            either be a remote server (e.g. https://) or a local
+                            file on the filesystem (e.g. file://). Often the
+                            metadata file contains all of the required
+                            properties for SAML authentication and the other
+                            parameters are not required.</para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term><property>saml-idp-url</property></term>
+                    <listitem>
+                        <para>The base URL of the SAML IdP. This is the URL that
+                            the SAML authentication extension will use to redirect
+                            when requesting SAML authentication. If the
+                            <property>saml-idp-metadata</property> property is
+                            provided, this parameter will be ignored. If the metadata
+                            file is not provided this property is required.</para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term><property>saml-entity-id</property></term>
+                    <listitem>
+                        <para>The entity ID of the Guacamole SAML client, which is
+                            generally the URL of the Guacamole server, but is not
+                            required to be so. This property is required if
+                            either the <property>saml-idp-metadata-url</property>
+                            property is not specified, or if the provided
+                            metadata file does not contain the SAML SP Entity
+                            ID for Guacamole Client.</para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term><property>saml-callback-url</property></term>
+                    <listitem>
+                        <para>The URL that the IdP will use once authentication has
+                            succeeded to return to the Guacamole web application and
+                            provide the authentication details to the SAML extension.
+                            The SAML extension currently only supports callback as a
+                            POST operation to this callback URL. This property
+                            is required if either the
+                            <property>saml-idp-metadata-url</property> property
+                            is not specified, or if the provided metadata file
+                            does not contain the SAML SP Entity ID for
+                            Guacamole Client.</para>
```

Comment:
I believe this property is still always required, as it drives the redirect issued from within the callback REST endpoint.

It's conceivable that it _could_ be made optional, with the redirect derived from the ACS URL, though I'm not sure how reasonable that will be to implement.

Reply:
Ah, okay, I see.